### PR TITLE
Support python events

### PR DESCRIPTION
**Pull request by prateeksultania**

### CODE REVIEW

File: modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java [hunk 1]
```diff
@@ -278,8 +278,10 @@ private Model getParent(Model model) {
                     }
                 });
 
+                Iterator<String> iter = modelKeys.iterator();
                 // process models only
-                for (String name : modelKeys) {
+                while (iter.hasNext()) {
```
Comment (rishabhja) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java]: I think we do not need an iterator here. Can we remove this?

File: modules/swagger-codegen/src/main/resources/python/events_handler_client.mustache [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        self.{{eventHandlerParamName}} = {{eventHandlerParamName}}
+
+    def handle(self, env) :
+        request_body_size = int(env.get('CONTENT_LENGTH', 0))
+        event_dict = env['wsgi.input'].read(request_body_size).decode("utf-8")
+        event = Event(event_dict)
+        payload = jwt.decode(env['HTTP_X_FLOCK_EVENT_TOKEN'], self.app_secret, algorithm='HS256')
+        if payload['userId'] == event.userId:
+            {{#events}}
+            {{#isFirst}}
```
Comment (rishabhja) [inline modules/swagger-codegen/src/main/resources/python/events_handler_client.mustache]: Can we create a map of events  to methods instead?

**Comment by prateeksultania** [comment]
GTM if tested

**Comment by rishabhja** [comment]
We can close this pull request now as "support_javascript_events" has been merge with this branch

**Comment by prateeksultania** [comment]
Have merged both the PRs in order. First merged this one and then support_javascript_events.

**Pull request merged**